src/Mod/PartDesign/App/FeatureHelix.cpp

```diff
@@ -229,13 +229,14 @@ App::DocumentObjectExecReturn* Helix::execute()
         if ( UseMakePipe.getValue() )  {
             TopoDS_Shape path = generateHelixPath(0.0, 1.0);
             TopoDS_Shape face = Part::FaceMakerCheese::makeFace(wires);
+            face.Move(invObjLoc);
             BRepOffsetAPI_MakePipe mkPS(TopoDS::Wire(path), face, GeomFill_Trihedron::GeomFill_IsFrenet, Standard_False);
 
             mkPS.Build();
 
             result = mkPS.Shape();
         } else {
-            /*  @deprecrated  12/01/2023  This is legacy code, here to maintain backward compatibility with existing models without altering their
+            /*  Deprecated  12/01/2023  This is legacy code, here to maintain backward compatibility with existing models without altering their
```

Comment:
Why is the doxygen formatting being altered here ?

Reply:
Uh, that was me correcting my typo of a line I put there, I thought.   No doxygen involved.  Note the extra 'r' as well as the @ were both me, earlier.

Reply:
Oh. I see. Thanks for the clarification. In that case perhaps keep the @deprecated Eventually we'll get our act together and doxy up the rest of the code. Might as well start.